lte/gateway/c/oai/tasks/amf/nas5g_network.h

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <sstream>
+#include "amf_config.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "bstrlib.h"
+#ifdef __cplusplus
+};
+#endif
+
+using namespace std;
+namespace magma5g {
+// Free up the bstring
+void bdestroy_wrapper(bstring* b);
+
+// Free up ptr
+void free_wrapper(void** ptr);
```

Comment:
Why are these getting duplicated? Can we not use the free_wrapper defined in `dynamic_memory_check.c` ?

Reply:
resolved